Index: .idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/vcs.xml b/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1690532482584)
+++ b/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/vcs.xml	(date 1690532482584)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/indexLayout.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/indexLayout.xml b/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/indexLayout.xml
new file mode 100644
--- /dev/null	(date 1690532482651)
+++ b/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/indexLayout.xml	(date 1690532482651)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="UserContentModel">
+    <attachedFolders />
+    <explicitIncludes />
+    <explicitExcludes />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/projectSettingsUpdater.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/projectSettingsUpdater.xml b/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/projectSettingsUpdater.xml
new file mode 100644
--- /dev/null	(date 1690527665197)
+++ b/.idea/.idea.Proyecto Avanzada Web II Cuatrimestre 2023/.idea/projectSettingsUpdater.xml	(date 1690527665197)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RiderProjectSettingsUpdater">
+    <option name="vcsConfiguration" value="2" />
+  </component>
+</project>
\ No newline at end of file
